sig-architecture/production-readiness.md

```diff
@@ -5,21 +5,90 @@ Kubernetes are observable, scalable and supportable, can be safely operated in
+the PRR. The template points out these as needed.
+
+## Submitting a KEP for production readiness approval
+
+The KEP template production readiness questionnaire should be filled out by the
+KEP authors, and reviewed by the SIG leads. Once the leads are satisfied with
+both the overall KEP (i.e., it is ready to move to `implementable` state) and
+the PRR answers, the authors may request PRR approval:
+
+* Assign a PRR approver from the `prod-readiness-approvers` list in the
+  [OWNERS_ALIASES] file. This may be done earlier as well, to get early feedback
+  or just to let the approver know. Reach out on the `#prod-readiness` Slack
+  channel or just pick someone from the list. The team may rebalance the
+  assignees if necessary.
+* Update the `kep.yaml`, setting the `stage`, `latest-milestone`, and the
+  `milestone` struct (which captures per-stage release versions).
+* Create a `prod-readiness/<sig>/<KEP number>.yaml` file, with the PRR
+  approver's GitHub handle for the specific stage
+* See this [example PRR approval request PR].
+
+The PRR approvers use the `kepctl` tool to identify
+all outstanding requests for PRR approval, and are responsible for providing
+timely feedback so that KEPs are not delayed. If your need is urgent it doesn't
+hurt to ping the person on Slack, as well, but it is not necessary.
+
+## Common feedback from reviewers
+
+Some common issues we see:
+* Missing metrics. Often metrics are overlooked, or sometimes they are defined
+  but not wired up (implemented). Please see this [example metrics PR] to better
+  understand how to add a metric.
+* Be sure to differentiate between when to use a *metric* vs an *event*. Metrics
+  are for the cluster operators and should be primarily about system status and issues,
+  whereas events can be used to surface user errors. That is, in general the audience
+  for metrics is cluster operator whereas the audience for events is the user (creator
+  of that resource). Users won't usually see metrics, and cluster operators
+  don't generally need to know about user mistakes.
+
+## Becoming a prod readiness reviewer or approver
+
+The prod readiness team is open and eager to add new members. The ideal
+production readiness approver is someone with deep knowledge of Kubernetes overall
+and the ability to think from the point of view of a cluster operator. An
+excellent background would, for example, be someone that is an SRE for a fleet
+of Kubernetes clusters.
+
+To become a reviewer:
+ * Inform the PRR team on Slack (`#prod-readiness`) or by attending the
+   bi-weekly meeting.
+ * Read/study previous PRR comments and production readiness responses in existing KEPs.
+ * Choose some KEPs requiring PRR and perform a review. Put "shadow prod readiness review"
+   in your review comments so that the assigned PRR approver knows your intent.
+ * After two release cycles, if you have shown good judgement and quality reviews,
```

Comment:
Should this be "at least one release"? If we move to 3 releases/yr it will take 8mo of shadowing to make someone an approver, which seems unusually high for k8s, especially on a brand new team